### ISSUE TITLE
Add a way to price items through Backpack.tf Classifieds, instead of prices.tf

### ISSUE THREAD

**Issue by CrustTF2** [TF2Autobot/tf2autobot#249]
**Is your feature request related to a problem? Please describe.**
 
Sort of. Prices.tf sometimes messes up its prices and makes your bot pay WAY over buy orders for items, mostly unusuals (for instance, my bot ended up buying a Stormy Storm Brown Bomber for 28 Keys 58 Refined, when all other bot buy orders were at 20). Having a way to price your items through classifieds/match classified listings would partially fix this issue and would allow users to have reliable unusual bots without having to pay say 10 keys a month for a bot service. Granted, the issue would be risky and be subject to price manipulators, but is way safer than paying 8-10 keys over other buyers and getting screwed over on a hat.

**Describe the feature you'd like**

Have an option for your bot to price items based off of classified listings, like buy orders on unusuals, instead of relying entirely on prices.tf. Have it match the prices of the highest buyer/lowest seller. This would be a great option/alternative for people that use their bots to bank unusuals.

**Describe alternatives you've considered**
There are only 3 alternatives that I know of:
1. Manually price all unusuals: People have told me to do this but it's simply impossible to say manually price 300+ unusuals and expect it to keep up with other bots' listings, since they have overcut/undercut scripts.
2. Use prices.tf autopricing: As stated before, this can be extremely risky or just flat out screw you over. Most of the time the autopricer prices items way too high for what they are currently at, and it can buy way above other buyers and leave you with barely any profit or at a 1+ key loss.
3. Pay for a bot service: This is the only other option that I can think of for having a reliable unusual banking bot, but not everyone can afford the costs (for instance, gladiator.tf charges 10 keys a month for its bot with the overcut/undercut script).

**Comment by idinium96** [comment]
Thanks for the suggestion.

**Comment by Dex-Spirit** [comment]
There should be requirements to update the price (to avoid price manipulation):
+ there should be at least 3 listing with same price or price difference shouldn’t be more than 0.33 ref
+ these listings should have been staying there for 5-30 min

**Comment by rockeyroad1415** [comment]
I agree with this, if there is someway to implement this into v4 that would be great. Can help with logic.
